### ISSUE TITLE
Sampling only noise when predicting x_0 directly

### ISSUE THREAD

**Issue by coldow95** [lucidrains/denoising-diffusion-pytorch#218]
Hi everyone!,

I am currently trying to use x_0-prediction and DDIM sampling. However, while the loss decreases successfully, I can only sample random images, no matter how long the model was trained. 

When I use x_noise prediction, the model samples correctly. 

Are there any settings that I need to change or has anyone encountered this problem before?

Best regards!
